extensions/odoo_theme/static/js/layout.js

```diff
@@ -8,24 +8,26 @@
             // Enforce the presence of the `img-fluid` class on all images.
             image.classList.add('img-fluid');
 
-            // Add a modal to each image.
-            const modalContainer = document.createElement('div');
-            modalContainer.classList.add('o_image_modal');
-            modalContainer.innerHTML = `<div class="modal fade" id="modal-${imageModalId}">
-                   <div class="modal-dialog modal-dialog-centered">
-                     <div class="modal-content">
-                       <div class="modal-body">
-                         <img src="${image.src}" alt="${image.alt}"/>
-                         <button type="button" class="btn-close btn-close-white" data-bs-dismiss="modal" aria-label="Close"></button>
+            // Add a modal to each image that has no target.
+            if (image.parentElement.tagName !== 'A') {
+                const modalContainer = document.createElement('div');
+                modalContainer.classList.add('o_image_modal');
+                modalContainer.innerHTML = `<div class="modal fade" id="modal-${imageModalId}">
+                       <div class="modal-dialog modal-dialog-centered">
+                         <div class="modal-content">
+                           <div class="modal-body">
+                             <img src="${image.src}" alt="${image.alt}"/>
```

Comment:
This was already the case before, but the `alt` text is only for screenreaders. Yet in our case, it would be useful as a `title` too, because it's something everybody should see.

Could we add that too?

BTW, `img.src` and `image.alt` are data/text, you can't inject that safely inside HTML markup, they should be escaped.
Fortunately you have `$u.escape()`  ;-)

Reply:
I'm not sure it would be a good idea to set the original image's `alt` as `title` of the modal image. As you said, `alt` texts are meant for screenreaders, so they're usually written as a description of what the user sees on screen (e.g., "A sales order with two order lines."). I don't think that it has any added value to show that on-screen too.